### PR TITLE
Rollback google auth

### PR DESCRIPTION
**Pull request by qiwzhang**
As google-auth doesn't support python2.7 any more. 
But ESP has to stay with python2.7 since its bazel stay at 0.21.

### CODE REVIEW

**Comment by qiwzhang** [comment]
It passed pre-e2e test.  https://endpoints-jenkins.appspot.com/job/jenkins-testing/job/pr-e2e/286/

File: Jenkinsfile [hunk 1]
```diff
@@ -52,7 +52,7 @@ SUPPORTED_STAGES = [
 ]
 
 // Supported VM Images
-SLAVE_IMAGE = 'gcr.io/endpoints-jenkins/debian-9:0.12'
+SLAVE_IMAGE = 'gcr.io/endpoints-jenkins/debian-9:0.13'
```
Comment (nareddyt) [inline Jenkinsfile:55]: Why does this need to be updated?

Comment (TAOXUY) [reply Jenkinsfile:55]: Seems the environment installed in script/linux-install-software should be included in this image, which will be used in jenkins.

Comment (qiwzhang) [reply Jenkinsfile:55]: Good question.  My impression is jenkins slave image is specified here.  But actually, it is in the jenkins endpoint config.  Well, to be safe, also update it here.

**Pull request merged**